src/main/java/omero/gateway/model/TableDataColumn.java

```diff
@@ -91,7 +92,9 @@ public TableDataColumn(String name, String description, int index,
         }
         if (!(type.equals(Object.class) || type.equals(Boolean.class)
                 || type.equals(Double.class) || type.equals(Double[].class)
-                || type.equals(FileAnnotationData.class)
+                || type.equals(OriginalFile.class)
+                || type.equals(FileAnnotationData.class) // Deprecated!
+                // || type.equals(DatasetData.class) // doesn't work
```

Comment:
This was previously passing. So that is a breaking change. Saying that the implementation was not working so probably nobody used it otherwise we will have had a bug report